packages/shared/src/components/CombatReport/index.tsx

```diff
@@ -143,6 +146,16 @@ export const CombatReportInternal = ({ matchId, roundId }: { matchId: string; ro
             Scoreboard
           </a>
         )}
+        {clientContext.isDesktop && viewerIsOwner && (
+          <a
+            className={`tab ${activeTab === 'video' ? 'tab-active' : ''}`}
+            onClick={() => {
+              setActiveTab('video');
+            }}
+          >
+            Video
```

Comment:
no changes needed for now but I wonder if this should be combined with replay.. have you thought about it?

Reply:
yeah we'll definitely improve it to utilize more of the replay/logged events -- saving that for future PRs I think since this one is already quite large